src/test/java/com/google/sps/data/BackgroundTaskManagerTest.java

```diff
@@ -59,4 +63,11 @@ public void replaceFaultyInstancesTest() {
         Assert.assertEquals(e.getMessage(), "Unimplemented");
     }
   }  
+  @Test
+  public void deleteInactiveAttendeesTest() {
+    backgroundManager.deleteInactiveAttendees(datastore,sessionId);
+    Assert.assertEquals(datastore.getAttendeesInSession, {"Taniece", "Chris"});
```

Comment:
does this work?

Reply:
also i think you reversed this, you expect "Taniece" and "Chris" but you actually get datastore.getAttendeesInSession(parameters)